plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetWriterConfig.java

```diff
@@ -72,6 +73,20 @@ public ParquetWriterOptions toParquetWriterOptions()
         return ParquetWriterOptions.builder()
                 .setMaxBlockSize(getBlockSize())
                 .setMaxPageSize(getPageSize())
+                .setRowGroupMaxRowCount(getRowGroupMaxRowCount())
                 .build();
     }
+
+    @Config("parquet.writer.row-group-max-row-count")
+    @ConfigDescription("Maximum number of rows written to a single Parquet row group")
```

Comment:
Do we know why io.trino.parquet.writer.ParquetWriterOptions#maxRowGroupSize isn't exposed in this class?

Reply:
The naming isn't consistent, `maxRowGroupSize` is mapped to `ParquetWriterConfig#getBlockSize`, I can't tell if it was intentional or not: https://github.com/trinodb/trino/pull/3400/files#diff-6a8c23f3ad5b6f11547031b4f8aa47593d82e70f611a4704ff0b27f1eca0cfb1